lld/ELF/Arch/RISCV.cpp

```diff
@@ -1533,3 +1540,21 @@ template <class ELFT> void RISCV::scanSection1(InputSectionBase &sec) {
 void RISCV::scanSection(InputSectionBase &sec) {
   invokeELFT(scanSection1, sec);
 }
+
+namespace lld::elf {
+uint32_t getRISCVVendorRelType(StringRef rvVendor) {
```

Comment:
Nit: This could maybe be something like the following for clarity, given it now doesn't return a type:

```suggestion
uint32_t getRISCVVendorRelMarker(StringRef rvVendor) {
```